Presentation.Web/Models/External/V2/Response/Organization/OrganizationUserResponseDTO.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Presentation.Web.Models.External.V2.Types;
+
+namespace Presentation.Web.Models.External.V2.Response.Organization
+{
+    public class OrganizationUserResponseDTO : IdentityNamePairResponseDTO
+    {
+        /// <summary>
+        /// User's first name
+        /// </summary>
+        public string FirstName { get; }
+        /// <summary>
+        /// User's last name(s)
+        /// </summary>
+        public string LastName { get; }
+        /// <summary>
+        /// User's email
+        /// </summary>
+        public string Email { get; }
+        /// <summary>
+        /// User's phone number
+        /// </summary>
+        public string PhoneNumber { get; }
+        /// <summary>
+        /// Determines if the user credentials can be used to request a KITOS API token
+        /// </summary>
+        public bool APIAccess { get; }
+        /// <summary>
+        /// Roles assigned to the user within the context of the organization
+        /// </summary>
+        public IEnumerable<OrganizationUserRole> Roles { get; }
+
+        public OrganizationUserResponseDTO(Guid uuid, string name, string firstName, string lastName, string email, bool apiAccess, string phoneNumber, IEnumerable<OrganizationUserRole> roles)
```

Comment:
Man kunne måske være flink at skrive "fullName" i stedet for "name" i denne context. Så ved man hvad der bliver givet videre til IdentityNamePairResponseDTO delen

Reply:
Nå det viser sig at "fullName" gør det umuligt at få deserializeren til at virke korrekt fordi den så vil lede efter et argument til ctor der hedder fullName i json kontrakten (og det hedder jo "name" grundet base class". Jeg namer tilbage.

Reply:
nah jeg fjerner ctor... så er der ikke det problem